tests/unit/test_download.py

```diff
@@ -121,6 +128,40 @@ def test_get_final_io_task(self):
             self.assertEqual(f.read(), ref_contents)
 
 
+class TestDownloadSeekableOutputManager(BaseDownloadOutputManagerTest):
+    def setUp(self):
+        super(TestDownloadSeekableOutputManager, self).setUp()
+        self.download_output_manager = DownloadSeekableOutputManager(
+            self.osutil, self.transfer_coordinator)
+
+        # Create a fileobj to write to
+        self.fileobj = open(self.filename, 'wb')
+
+        self.call_args = CallArgs(fileobj=self.fileobj)
+        self.future = self.get_transfer_future(self.call_args)
+
+    def tearDown(self):
+        self.fileobj.close()
+        super(TestDownloadSeekableOutputManager, self).tearDown()
+
+    def test_is_compatible(self):
```

Comment:
Could we get some tests for incompatible types, for instance a non-seekable `io.IOBase`